graphistry/compute/cluster.py

```diff
@@ -391,9 +392,9 @@ def _transform_dbscan(
 
             emb = None
             if umap and cols is None:
-                emb, X, y = res.transform_umap(df, ydf, kind=kind, return_graph=False)
+                emb, X, y = res.transform_umap(df, ydf, kind=kind, return_graph=False)  # type: ignore
             else:
-                X, y = res.transform(df, ydf, kind=kind, return_graph=False)
+                X, y = res.transform(df, ydf, kind=kind, return_graph=False)  # type: ignore
```

Comment:
any reason we need these type ignores? i had done some work recently to make `transform()` type so am surprised

Reply:
I thought I replied somewhere transform can return a Plottable

`return_graph=False` needs some sort of overloading if it is possible